tests/test_processing_stack.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import time
+
+from pytest import mark
+
+APPLICATION_TAG_KEY = "ApplicationName"
+APPLICATION_TAG_VAL = "geospatial-data-lake"
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# TODO: get resources from correct AWS stack (DEPLOY_ENV), if multiple stacks are deployed to
+# single AWS account
+ENV = os.environ.get("DEPLOY_ENV", "dev")
+
+
+@mark.infrastructure
+def test_should_return_healthy_compute_environment(batch_client):
+    datalake_compute_environment = {}
+
+    response = batch_client.describe_compute_environments()
+    for compute_environment in response["computeEnvironments"]:
+        if (
+            APPLICATION_TAG_KEY in compute_environment["tags"].keys()
+            and compute_environment["tags"][APPLICATION_TAG_KEY] == APPLICATION_TAG_VAL
+        ):
+            datalake_compute_environment = compute_environment
+            logger.info("Compute Environment: %s", datalake_compute_environment)
+            break
+
+    assert (
+        datalake_compute_environment["state"] == "ENABLED"
+        and datalake_compute_environment["status"] == "VALID"
+        and datalake_compute_environment["statusReason"] == "ComputeEnvironment Healthy"
+    )
+
+
+@mark.infrastructure
+def test_should_prove_compute_environment_is_working_by_running_single_job(batch_client):
+    datalake_job_definition = {}
+    datalake_job_queue = {}
+
+    job_definitions_response = batch_client.describe_job_definitions()
+    for job_definition in job_definitions_response["jobDefinitions"]:
+        if (
+            APPLICATION_TAG_KEY in job_definition["tags"].keys()
+            and job_definition["tags"][APPLICATION_TAG_KEY] == APPLICATION_TAG_VAL
+        ):
+            datalake_job_definition = job_definition
+            logger.info("Job Definition: %s", datalake_job_definition)
+            break
```

Comment:
I would pull this out into a separate function. That way you can `raise NoJobDefinitionFound` or something if none of the job definitions match (avoiding a `KeyError` when running `datalake_job_definition["jobDefinitionName"]` below), and you won't need the `datalake_job_definition = {}` above.

Reply:
Resources detection pulled out to functions in ec554077d939ab1f1e62e0ebdc5cf68a550c400d